packages/fe/store/datasets.js

```diff
@@ -18,7 +14,8 @@ const state = () => ({
   basicStats: false,
   filters: false,
   sort: false,
-  limit: false
+  limit: false,
```

Comment:
Looks like you have 2 limit values set in the store?

Reply:
done